client/src/services/ApiService.js

```diff
@@ -160,11 +160,11 @@ class ApiService {
     return list;
   }
 
-  async addRelationship(sourceId, targetId, relationshipType, relationshipId) {
+  async addRelationship(sourceId, targetId, relationshipType, relationshipId, properties) {
     await this.initialize();
 
     return await this.client.upsertRelationship(sourceId, relationshipId,
-      { $relationshipName: relationshipType, $targetId: targetId });
+      { $relationshipName: relationshipType, $targetId: targetId, ...properties });
```

Comment:
I would suggest putting `...properties` at the start of the object.

In the current code, if the user specifies $relationshipName as a property, then that would overwrite our property.

Reply:
Done